frontend/controller/actions/identity.js

```diff
@@ -408,8 +409,10 @@ export default (sbp('sbp/selectors/register', {
           return sbp('okTurtles.eventQueue/queueEvent', `login:${identityContractID ?? '(null)'}`, ['chelonia/contract/sync', ids, { force: true }])
         }))
       } catch (err) {
-        alert(L('Sync error during login: {msg}', { msg: err?.message || 'unknown error' }))
         console.error('Error during contract sync upon login (syncing all contractIDs)', err)
+
+        const humanErr = L('Sync error during login: {msg}', { msg: err?.message || 'unknown error' })
+        throw new GIErrorUIRuntimeError(humanErr)
```

Comment:
In cases where you `throw`, you should use the `cause` option including the original error (see example below). This allows error handlers access to the original error, which can be helpful for error handling and debugging.

```
throw new GIErrorUIRuntimeError(humanErr, { cause: err })
```